algebird-test/src/main/scala/com/twitter/algebird/BaseProperties.scala

```diff
@@ -284,7 +284,7 @@ object BaseProperties extends MetricProperties {
     isDistributiveDifferentTypes[T, T]
 
   def timesIsAssociative[T: Ring: Arbitrary: Equiv]: Prop =
-    'timesIsAssociative |: forAll { (a: T, b: T, c: T) =>
+    Symbol("timesIsAssociative") |: forAll { (a: T, b: T, c: T) =>
```

Comment:
I think you can just use strings here actually. No need for the Symbols at all.

Reply:
indeed! I wasn't sure if there was a not obvious reason for the saved up allocations.